internal/organizer/db/companies_part3.go

```diff
@@ -4,43 +4,1138 @@ import "github.com/readytotouch/readytotouch/internal/domain"
+				OverviewURL: "",
+				ReviewsURL:  "",
+				Verified:    false,
+			},
+			OttaProfileSlug:   "Cloudflare-1",
+			YouTubeChannelURL: "",
+			GoMainLanguage:    false,
+			Vacancies: domain.Vacancies{
+				domain.Go: nil,
+				domain.Rust: []string{
+					"https://www.linkedin.com/jobs/view/4041774900/",
+				},
+				domain.Zig:     nil,
+				domain.Scala:   nil,
+				domain.Elixir:  nil,
+				domain.Clojure: nil,
+				domain.Haskell: nil,
+			},
+			ShortDescription: "Web application security and performance.",
+			Industries:       []domain.Industry{},
+			HasEmployeesFromCountries: []domain.Country{
+				domain.Ukraine,
+				domain.Czechia,
+			},
+		},
+
+		// Smart Electric Vehicle charging stations
+		{
+			ID:   0,  // system
+			Type: "", // system
+			Name: "EverCharge ",
+			URL:  "https://evercharge.com/",
+			LinkedInProfile: domain.LinkedInProfile{
+				ID:       4817778,
+				Alias:    "evercharge",
+				Name:     "EverCharge ",
+				Verified: true,
+			},
+			GitHubProfile: domain.GitHubProfile{
+				Login:             "",
+				GoRepositoryCount: 0,
+				Verified:          false,
+			},
+			GlassdoorProfile: domain.GlassdoorProfile{
+				OverviewURL: "",
+				ReviewsURL:  "",
+				Verified:    false,
+			},
+			OttaProfileSlug:   "",
+			YouTubeChannelURL: "",
+			GoMainLanguage:    false,
+			Vacancies: domain.Vacancies{
+				domain.Go: nil,
+				domain.Rust: []string{
+					"https://www.linkedin.com/jobs/view/4040225807/",
+				},
+				domain.Zig:     nil,
+				domain.Scala:   nil,
+				domain.Elixir:  nil,
+				domain.Clojure: nil,
+				domain.Haskell: nil,
+			},
+			ShortDescription: "Smart Electric Vehicle charging stations",
+			Industries:       []domain.Industry{},
+			HasEmployeesFromCountries: []domain.Country{
+				domain.Ukraine,
+			},
+		},
+
+		// VPN company - ExpressVPN
+		{
+			ID:   0,  // system
+			Type: "", // system
+			Name: "ExpressVPN ",
+			URL:  "https://www.expressvpn.com/",
+			LinkedInProfile: domain.LinkedInProfile{
+				ID:       14488856,
+				Alias:    "expressvpn",
+				Name:     "ExpressVPN",
+				Verified: false,
+			},
+			GitHubProfile: domain.GitHubProfile{
+				Login:             "expressvpn",
+				GoRepositoryCount: 3,
+				Verified:          true,
+			},
+			GlassdoorProfile: domain.GlassdoorProfile{
+				OverviewURL: "",
+				ReviewsURL:  "",
+				Verified:    false,
+			},
+			OttaProfileSlug:   "ExpressVPN",
+			YouTubeChannelURL: "",
+			GoMainLanguage:    false,
+			Vacancies: domain.Vacancies{
+				domain.Go: []string{
+					"https://www.linkedin.com/jobs/view/4058715430/,",
+				},
+				domain.Rust:    nil,
+				domain.Zig:     nil,
+				domain.Scala:   nil,
+				domain.Elixir:  nil,
+				domain.Clojure: nil,
+				domain.Haskell: nil,
+			},
+			ShortDescription:          "VPN company",
+			Industries:                []domain.Industry{},
+			HasEmployeesFromCountries: []domain.Country{},
+		},
+
+		//GlobalLogic -  full-lifecycle product development services
```

Comment:
It's outsource company,This is an outsourcing company, and we’re looking for product companies and startups for direct employment, without intermediaries